src/main/java/io/cdap/plugin/gcp/bigquery/action/BigQueryExecute.java

```diff
@@ -279,6 +289,10 @@ public String getSql() {
       return sql;
     }
 
+    public Boolean getStoreResults() {
```

Comment:
I don't see `getStoreResults` is used at backend, is it purely used for UI ? Do you need to make sure only when `getStoreResults` is true, you store the result in the table ?

Reply:
addressed it in `review comments` commit https://github.com/data-integrations/google-cloud/pull/1042/files#diff-9ccb1a50b6e5a7a6b2c016c6aaa7605605d66e7e6bfb5e4e95a16ac7c35caa89R118